lnd_test.go

```diff
@@ -898,47 +930,39 @@ func TestLightningNetworkDaemon(t *testing.T) {
 	// drive blockchain related events within the network.
 	btcdHarness, err := rpctest.New(harnessNetParams, handlers, nil)
 	if err != nil {
-		t.Fatalf("unable to create mining node: %v", err)
+		ht.Fatalf("unable to create mining node: %v", err)
 	}
 	defer btcdHarness.TearDown()
 	if err := btcdHarness.SetUp(true, 50); err != nil {
-		t.Fatalf("unable to set up mining node: %v", err)
+		ht.Fatalf("unable to set up mining node: %v", err)
 	}
 	if err := btcdHarness.Node.NotifyNewTransactions(false); err != nil {
-		t.Fatalf("unable to request transaction notifications: %v", err)
+		ht.Fatalf("unable to request transaction notifications: %v", err)
 	}
 
 	// With the btcd harness created, we can now complete the
 	// initialization of the network. args - list of lnd arguments,
 	// example: "--debuglevel=debug"
 	// TODO(roasbeef): create master balanced channel with all the monies?
 	if err := lndHarness.InitializeSeedNodes(btcdHarness, nil); err != nil {
-		t.Fatalf("unable to initialize seed nodes: %v", err)
+		ht.Fatalf("unable to initialize seed nodes: %v", err)
 	}
 	if err = lndHarness.SetUp(); err != nil {
-		t.Fatalf("unable to set up test lightning network: %v", err)
+		ht.Fatalf("unable to set up test lightning network: %v", err)
 	}
 
-	t.Logf("Running %v integration tests", len(testCases))
-	for name, test := range testCases {
-		errChan := ht.RunTest(lndHarness, test)
-
-		select {
-		// Attempt to read from the error channel created for this
-		// specific test. If this error is non-nil then the test passed
-		// without any problems.
-		case err := <-errChan:
-			if err != nil {
-				t.Fatalf("Fail: (%v): exited with error: \n%v",
-					name, err)
-			}
-			t.Logf("Passed: (%v)", name)
+	// Watch for lnd error to occur and fatal if we receive one,
```

Comment:
I think this comment would read more clearly if it was slightly reworded as: 
> Spawn a new goroutine to watch for any fatal errors that any of the running lnd processes encounter. If an error occurs, then the test fails immediately with a fatal error. 

If we swap the comment to the version above, do we lose any semantics you intended to convey in the original process?